### ISSUE TITLE
Подсветка измененных свойств палитре

### ISSUE THREAD

**Issue by Daabramov** [marmyshev/edt-plugins#7]
В палитре свойств показывать\подсвечивать измененные значения по сравнению со значением "по-умолчанию"
![image](https://user-images.githubusercontent.com/31928832/109431033-7faedc00-7a15-11eb-9e9f-650f36b65056.png)
![image](https://user-images.githubusercontent.com/31928832/109431058-9ce3aa80-7a15-11eb-9344-6e6f53f5369a.png)

**Comment by zeegin** [comment]
Вот это мегапозеный плагин. И еще кнопочку - вернуть дефаулт, пожалуцста.

**Comment by zeegin** [comment]
И еще можно фильтр в палитре "показать только измененные".

**Comment by KovAlexey** [comment]
А это вообще реально то?

**Comment by zeegin** [comment]
> А это вообще реально то?

Сравнить значения формы со значениями по умолчанию?

**Comment by KovAlexey** [comment]
> Сравнить значения формы со значениями по умолчанию?

Нет, как либо воздействовать на элементы панели более менее адекватными способами

**Comment by tormozit** [comment]
Че то я смотрю тут плагинописание процветает прям. Неужели все так плохо?
